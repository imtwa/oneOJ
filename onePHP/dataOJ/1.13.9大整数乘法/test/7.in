3154241425572451533536364264463856843576215663587852263384275521316117267557281576121535528142237555738523347336412872558745744325554568164151712675211436481636711886865811128616451555744536
3532744518125341472878756666262852847288867278622615475886116243522728753535238258585551881437287162137245563571483124543357263848814744265872238165513332332867435565273248352671818572187753