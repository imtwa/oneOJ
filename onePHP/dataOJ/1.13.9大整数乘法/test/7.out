11143129105034940433017724206522133726521973786182017545156023053459508707248891538892288705534801651829441569352307545853034622257989847503640742183528820289302874425721626967752825437239647372318797419015707559655842280562260667162831015641413987344565859943107524491153741042057063135462776458959034867752507607647757810428678606482256157074400363461400051723269201268295867608